.buildkite/pipelines/dummy_pipeline.yml

```diff
@@ -0,0 +1,28 @@
+steps:
+  - label: "Dummy step"
+    key: "dummy"
+    commands:
+      - DEBIAN_FRONTEND=noninteractive && apt-get update && apt-get install -y curl 
+      - curl -L https://raw.githubusercontent.com/jdzsz/randomsleep/main/randomsleep.sh -o randomsleep.sh
```

Comment:
What if we add this script to the `monorepo` repository so we can maintain it if necessary and there is no need to install `curl` to fetch the script (it would be checked-out with the source code)?